deebot_client/command.py

```diff
@@ -94,11 +94,14 @@ async def execute(
         return False
 
     async def _execute(
-        self, authenticator: Authenticator, device_info: DeviceInfo, event_bus: EventBus
+        self, authenticator: Authenticator, device_info: DeviceInfo, event_bus: EventBus, rooms: list[Room] = []
     ) -> CommandResult:
         """Execute command."""
         response = await self._execute_api_request(authenticator, device_info)
 
+        if self.name == "getPos":
+            response["resp"]["body"]["data"]["rooms"] = rooms
```

Comment:
Why is this needed?
`getPos` specific code should go into the `GetPos` class and not in the general `Command` class.

Reply:
I don't know how to pass rooms parameter to GetPos class. This is the only way I've found. Will you give me a hand?

Reply:
I think this code should be in `map.py` and not in the command. The commands module is what we get from the API and we should not do calculations there.
Instead I think the calculation should be part of `map.py`, where you already have the rooms and the position